api/src/services/mail/index.ts

```diff
@@ -83,13 +84,35 @@ export class MailService {
 		return info;
 	}
 
-	private async renderTemplate(template: string, variables: Record<string, any>) {
+	private async getFirstExistingPath(paths: (string | null)[]): Promise<string | null> {
+		for (const path of paths) {
+			if (!path) {
+				continue;
+			}
+
+			if ((await fse.pathExists(path)) === true) {
+				return path;
+			}
+		}
+
+		return null;
+	}
+
+	private async renderTemplate(template: string, variables: Record<string, any>, language?: string | undefined | null) {
+		const localizedTemplatePath = language
+			? path.resolve(env['EMAIL_TEMPLATES_PATH'] as string, template + `.${language}.liquid`)
+			: null;
```

Comment:
Hi, thanks for response.
I understand that yet it might happen that for different languages especially for right to left you would like to have a but different views.

So with such solution we can have both, what you propose and different templates for different languages.

Also consider situation where for some languages you might want to have different colors of buttons or so.

Proposed implementation is not conflicting with your idea, just gives additional possibilities.

Reply:
Hi, thanks for response.
I understand that yet it might happen that for different languages especially for right to left you would like to have a but different views.

So with such solution we can have both, what you propose and different templates for different languages.

Also consider situation where for some languages you might want to have different colors of buttons or so.

Proposed implementation is not conflicting with your idea, just gives additional possibilities.